examples/basic/BUILD

```diff
@@ -27,5 +27,5 @@ csharp_nunit_test(
 csharp_library(
     name = "lib",
     srcs = ["lib.cs"],
-    target_frameworks = ["net472"],
+    target_frameworks = ["netcoreapp3.0"],
```

Comment:
This only works nicely if the TFMs match: if it were .NET standard the DLL would be in another folder.

I'm now thinking that for .NET Core, rather than wrapping with a customized assembly loader maybe we should just add more probing paths to the runtimeconfig.json. We'll still need a wrapper for .NET Framework I think (see #9 for more details).

So my goal is to change this library to `netstandardx.y` in the future.